client/src/components/Home/style.css

```diff
@@ -0,0 +1,33 @@
+.home {
+  margin: 0 auto;
```

Comment:
why do you need `margin: 0 auto;` if you use __flex__

Reply:
it works